specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -958,10 +958,76 @@
         "Succeeded",
         "Failed",
         "Cancelled",
-        "Cancelling"
+        "Cancelling",
+        "ValidationFailed"
       ],
       "type": "string"
     },
+    "ErrorCodeV2": {
+      "description": "Enums containing high level error codes.",
+      "enum": [
+        "InvalidRequest",
+        "InvalidArgument",
+        "InternalServerError",
+        "ServiceUnavailable",
+        "ResourceNotFound",
+        "Unauthorized",
+        "RequestRateTooHigh"
+      ],
+      "type": "string"
+    },
+    "InnerErrorV2": {
+      "description": "New Inner Error format which conforms to Cognitive Services API Guidelines which is available at https://microsoft.sharepoint.com/%3Aw%3A/t/CognitiveServicesPMO/EUoytcrjuJdKpeOKIK_QRC8BPtUYQpKBi8JsWyeDMRsWlQ?e=CPq8ow.\r\nThis contains required properties ErrorCode, message and optional properties target, details(key value pair), inner error(this can be nested).",
+      "type": "object",
+      "properties": {
+        "code": {
+          "format": "int32",
+          "description": "Gets detailed error code.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "error": {
```

Comment:
could you expand more on what `error` is?
According to the [guidelines](https://dev.azure.com/msazure/Cognitive%20Services/_wiki/wikis/Cognitive%20Services.wiki/15797/Errors-Long-running-operations), there is no `error` in `InnerError`, just `Code`, `Message`, `Target`, and `InnerError`